### DISCUSSION TITLE
ssl certificate expired

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [vercel/community#2224] [Help] [answered]
### Summary

My ssl vertificate expired 4 days ago, i follwed these instructions
https://vercel.com/guides/domain-not-generating-ssl-certificate
to check this result
https://dns.google/query?name=www.journeai.com&rr_type=CAA&ecs=
but am unsure what i am doing wrong?

Result for www.journeai.com/CAA with DNSSEC validation and without DNSSEC detail:
{
  "Status": 0 /* NOERROR */,
  "TC": false,
  "RD": true,
  "RA": true,
  "AD": false,
  "CD": false,
  "Question": [
    {
      "name": "www.journeai.com.",
      "type": 257 /* CAA */
    }
  ],
  "Answer": [
    {
      "name": "www.journeai.com.",
      "type": 5 /* CNAME */,
      "TTL": 3600,
      "data": "cname.vercel-dns.com."
    },
    {
      "name": "cname.vercel-dns.com.",
      "type": 257 /* CAA */,
      "TTL": 600,
      "data": "0 issue \"letsencrypt.org\""
    },
    {
      "name": "cname.vercel-dns.com.",
      "type": 257 /* CAA */,
      "TTL": 600,
      "data": "0 issue \"globalsign.com\""
    },
    {
      "name": "cname.vercel-dns.com.",
      "type": 257 /* CAA */,
      "TTL": 600,
      "data": "0 issue \"sectigo.com\""
    }
  ],
  "Comment": "Response from 2a00:edc0:6259:7:13::2."
}

**Comment by swarnava** [comment, marked as answer]
Hi, You need to add  www.journeai.com to the project

**Reply by 3a1b2c3** [reply]
What project exactly? in google? In vercel has been ok until a few days ago? now its NET::ERR_CERT_DATE_INVALID

This server could not prove that it is www.journeai.com; its security certificate expired 5 days ago. This may be caused by a misconfiguration or an attacker intercepting your connection. Your computer's clock is currently set to Saturday, April 29, 2023. Does that look right? If not, you should correct your system's clock and then refresh this page.

[Proceed to www.journeai.com (unsafe)](chrome-error://chromewebdata/#)

**Reply by 3a1b2c3** [reply]
Anyone please? Support has no reaction

**Reply by swarnava** [reply]
Our team has reached out to your mail box ending with @protonmail.com already couple of times:

>Thanks for reaching out to Vercel Support!
>
>I've had a look at your project and it appears www.journeai.com hasn't been added to your project settings, so there's no task to create this certificate.
>
>To add this domain to your project, you need to use the following instructions: https://vercel.com/docs/concepts/projects/domains/add-a-domain
>Once added, the certificate will then be generated and applied to your project.
>
>Please let us know if you have any further questions or issue
